### ISSUE TITLE
Result screen

### ISSUE THREAD

**Issue by dansvend** [equinor/attenuation-app#47]

**Comment by dansvend** [comment]
@KaterynaBasik This is ready for testing in `2024.719.78`.

**Comment by KaterynaBasik** [comment]
@dansvend User gets `Det oppstod en feil under testingen...` result when the attenuation is `0 dB` for at least one ear?

**Comment by dansvend** [comment]
> @dansvend User gets `Det oppstod en feil under testingen...` result when the attenuation is `0 dB` for at least one ear?

Correct. This error message will be shown in the following conditions:
- Missing volume results (which is practice will never happen)
- If the volume for one of the ears are 0
- If the volume for one the ears are higher without plugs than with plugs

**Comment by KaterynaBasik** [comment]
Tested (2024.719.78)

- [ ] The test results screen matches the design in Figma (exept of the grey line which as decided will not  be implemented in MVP)
- [ ] Depending on the test result, user receives one of the three possible result screens.
- [ ] All buttons function as expected.
- [ ] The colors of the text and graphic element correspond accurately to the test results
- [ ] `Test på nytt` pop up window with different options works as expected 

@dansvend With the build 2024.719.78 volume adjustment buttons don't work on iPad with headphones. But it works in the build 2024.712.62

**Comment by dansvend** [comment]
@KaterynaBasik I have a new build, `2024.723.83`, ready for testing with a potential fix for the problem with the volume adjustment buttons that didn't work on iPad with headphones 🧪

**Comment by KaterynaBasik** [comment]
> @KaterynaBasik I have a new build, `2024.723.83`, ready for testing with a potential fix for the problem with the volume adjustment buttons that didn't work on iPad with headphones 🧪

Tested. Volume adjustment works on iPad and iPhone now with and without headphones,

However I have noticed one more case that we probably need to take a look at.
User gets `Det oppstod en feil under testingen...` result also when the volume difference is too big, I guess. Do we have a best possible result? It looks like user gets `ikke godkjent `result  with` 0 dB` when the result is over the best possible result.
In case user presses 27 times down without plugs and then 5 times up with plugs they will get `Ikke godkjent ` result with `0 dB`

**Comment by dansvend** [comment]
> > @KaterynaBasik I have a new build, `2024.723.83`, ready for testing with a potential fix for the problem with the volume adjustment buttons that didn't work on iPad with headphones 🧪
> 
> Tested. Volume adjustment works on iPad and iPhone now with and without headphones,
> 
> However I have noticed one more case that we probably need to take a look at. User gets `Det oppstod en feil under testingen...` result also when the volume difference is too big, I guess. Do we have a best possible result? It looks like user gets `ikke godkjent `result with` 0 dB` when the result is over the best possible result. In case user presses 27 times down without plugs and then 5 times up with plugs they will get `Ikke godkjent ` result with `0 dB`

Thanks you so much for testing 💪 In the case you're describing we give `Ikke godkjent` because 1 measurement is at 0% system volume. The user then must have pretended to hear something even if they did not which should result in a `Ikke godkjent`. To get to 0% system volume the user needs to press the `-`-button 25 times. 1 press of a volume control button in the app adjust the system volume by 2%.
